.gitignore

```diff
@@ -12,3 +12,4 @@ scripts/release/packager/artifacts
 scripts/release/packager/build
 scripts/xcparty/.swiftpm
 MapboxMaps.xcodeproj
+Package~.swift
```

Comment:
I've never seen this before. What was in it?

Reply:
I saw it for the first time during this update, I'm not entirely sure why it is being produced all of a sudden. I'm still using Xcode 12.4 for my development. 

But it was carbon copy of the previous Package.swift before I updated dependency pointers.

Reply:
It looks like some kind of tmp file from some other editor. I'm hesitant to add it to the .gitignore. Could you revert this change, delete the file, and keep an eye out for whether it shows up again?